index.md

```diff
@@ -1,51 +1,61 @@
+- [**Event Streams**](glossary.md#event-streams): This is an IBM Cloud-based [Apache Kafka](https://kafka.apache.org/) managed service, the technology used for producing and consuming the data streams.
+- [**IBM Cloud Functions**](glossary.md#ibm-cloud-functions): Using these, Health Record Ingestion exposes a serverless RESTful Management API that is used to control and configure the system. 
+- [**Elasticsearch**](glossary.md#elasticSearch): This is the distributed NoSQL data store that is used to store information about batches.
 
-## Core Architecture
+## IBM Cloud dependencies  
+Health Record Ingestion was developed on the IBM Cloud and currently does not support running on other public or private clouds. However, as part of Project Alvearie, Health Record Ingestion is being transitioned into an open source project, with a goal of supporting public and private cloud deployments. The latest version has been pushed to GitHub. The team continues to work on moving all development activities into the open, and making Health Record Ingestion support other clouds. For more information about future plans, see the [Roadmap](roadmap.md).   
+
+## Core architecture
+<br>
+**Figure: Components of the Health Record Ingestion service**
+<br><br>
 ![core-architecture](assets/img/architecture-core.png)
+<br><br>
 
 ### Topics
-Health data, which may include [PHI](glossary.md#phi), is written to and read from the Kafka (IBM Event Streams) topics. There **_must be_** _separate topics for each tenant and Data Integrator_ in order to meet data separability requirements. 
+Kafka (IBM Event Streams) stores streams of records in categories or feeds called **topics**. Health data, which can include [Protected Health Information (PHI)](glossary.md#protected-health-information-phi), is written to and read from topics. To meet data separability requirements, **there must be separate topics for each tenant and Data Integrator**. 
 
-Additional topics may be created as desired. But, in general, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for [batch status notifications](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json). 
+As needed, you can create additional topics. Generally, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for batch status notifications. To review the relevant JSON for notifications, see the [API specification on GitHub](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json). 
```

Comment:
We have tried (albeit not very consistently) to not use 'you', which is why we had the prior. Should we  not do this?
I  don't think 'To review the relevant JSON for notifications, see the [API specification on GitHub](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json). ' is needed here. Just a link is fine.